src/wipplpy/modules/config_reader.py

```diff
@@ -0,0 +1,35 @@
+"""
+Extract from the local config the names of the servers and trees associated
+with the MDSplus databases for MST and BRB.
+"""
+
+import configparser
+import os
+
+
+class ConfigReader:
+
+    def __init__(self):
+        """
+        Read the server- and tree-names of BRB and MST into variables for easy
+        global access.
+        """
+        # Fetch the INI file located in the same directory
+        config_file = os.path.join(
+                os.path.dirname(__file__),
+                'mdsplus_config.ini')
+
+        config = configparser.ConfigParser() # initialize configparser
+        config.read(ini_file_path)
```

Comment:
We should check if the config file pointed to is a valid file with a `try` and `except`.